packages/webex/src/calling.js

```diff
@@ -68,6 +70,29 @@ class Calling extends EventEmitter {
       });
   }
 
+  async deregister() {
+    if (!this.registered) {
+      this.log.info('Authentication: deregister already done', logContext);
+
+      return Promise.resolve();
+    }
+
+    return (
+      // @ts-ignore
+      this.webex.internal.mercury
```

Comment:
Do we need to disconnect mercury as well with WDM deregistration?

Reply:
We connect mercury while registering. So, it made sense to get rid of the mercury connection as well every time we deregister.